frontend/jupyter/src/app/pages/form/form-new/form-protected-b/form-protected-b.component.html

```diff
@@ -1,9 +1,9 @@
-<lib-form-section title="Protected B">
+<lib-form-section title="Protected B" i18n-title>
   <div class="row">
     <div class="flex-column">
-      <mat-checkbox [formControl]="parentForm.get('prob')" i18n>
+      <mat-checkbox [formControl]="parentForm.get('prob')">
         <mat-icon matTooltip="Protected B" i18n-matTooltip> lock </mat-icon>
-        Run a Protected B notebook
+        <span i18n>&nbsp;Run a Protected B notebook</span>
```

Comment:
Why the non-breaking space?

Reply:
Styling reason. With the span surrounding the text for easy i18n declaration, there was no space between the icon and the text. The non breaking space is to replicate how it looked like before surrounding the text with the span.